lib/github-glue.ts

```diff
@@ -100,21 +101,52 @@ export class GitHubGlue {
      */
     public async addPRCc(pullRequestURL: string, cc: string):
         Promise<void> {
-        const id = cc.match(/(<.*>)/);
+        const id = cc.match(/<(.*)>/);
 
-        if (!id || id[1].match(/gitster@pobox\.com/)) {
+        if (!id || id[1] === "gitster@pobox.com") {
             return;
         }
 
+        const ccLower = id[1].toLowerCase();
         const url = GitGitGadget.parsePullRequestURL(pullRequestURL);
         const pr = await this.getPRInfo(url[0], url[2]);
-        const ccNow = pr.body.match(/\r?\n\r?\n(cc:.*)/is);
+        const trimBody = pr.body.trimRight();
+        const bodyEnd = trimBody.split(/\r?\n\s*?\r?\n/).pop() as string;
+
+        let found = false;
+        let footerSeparator = "\r\n";
+
+        if (bodyEnd.match(/:/)) try {
+            bodyEnd.split(/\r?\n/).reverse().forEach(line => {
+                const match = line.match(/^([a-z][-a-z0-9]+):\s*(.*)$/i);
+
+                if (!match) {       // stop if not a footer
+                    throw new Error("No Footer");
+                }
+
+                footerSeparator = ""; // body already has footers
+                if (match[1].toLowerCase() === "cc") try {
```

Comment:
<suggestion>
                 if (!found && match[1].toLowerCase() === "cc") try {
</suggestion>

Reply:
Pushed it up for one last round.